addons/curator/README.md

```diff
@@ -0,0 +1,12 @@
+## Curator
+
+Our adjustments and upgrades for Zeus.
+
+### Automatic Zeus assignment
+
+Some missions involve more than one Zeus. It is hard to get it working correctly and without problems when one Zeus disconnects. Not anymore. All units with variable `afm_curator_isZeus` set to true on init will have curator module assigned on startup (and JIP).
+
+## Authors
```

Comment:
I've noticed that I'm using `###` for that. 🤔

Reply:
`back_to_game` and `friendly_tracker` also use `##`, we could fix this in separate PR along with `requiredAddons` entry.

Reply:
Yeah, I agree. We'll do cleanup PR later. I also want to add `authors[] = {"<nicknames>"};` to CfgPatches later on.